packages/mwp-api-state/src/cache/util.js

```diff
@@ -66,5 +69,5 @@ export const cacheWriter = (cache: Cache) => (
 		// skip cache writing
 		return Promise.resolve(true);
 	}
-	return cache.set(JSON.stringify(query), response);
+	return cache.set(makeKey(memberId, query), response);
```

Comment:
Just to make sure I'm 💯 understanding the cache that this will be affecting. It's this IndexDB cache that will be key'd, correct? Jusstttt want to make sure there isn't another caching layer i'm not aware of... besides redux state. :)

![screen shot 2018-03-13 at 6 26 42 pm](https://user-images.githubusercontent.com/1933722/37373277-4d72b376-26ec-11e8-90d8-ab789a30f8ba.png)

Reply:
@eilinora I found the docs :) https://meetup.atlassian.net/wiki/spaces/WEG/pages/243499188/Caching

```The cache middleware implementation uses IndexedDB for local storage of API response data - it caches the JSON response rather than the full HTTP response in order to avoid complexity arising from the batched API request/response routine.```

Reply:
Excellent! Thank you!